src/components/Navbar/Navbar.tsx

```diff
@@ -1,5 +1,42 @@
+import styled from 'styled-components';
+import { useEffect, useState } from 'react';
+
+const PixelText = styled.span`
```

Comment:
can we shift this to root or tailwind config so that entire app can use the same font?

Reply:
Resolved